src/circuits/prfs/prfs.tcc

```diff
@@ -52,12 +18,11 @@ PRF_addr_a_pk_gadget<FieldT>::PRF_addr_a_pk_gadget(
         libsnark::pb_variable<FieldT>& a_sk,
         const std::string &annotation_prefix
       ) :
-      MiMC_hash_gadget<FieldT>(pb, {a_sk, get_zero(pb)}, get_iv_add(pb), annotation_prefix)
+      MiMC_hash_gadget<FieldT>(pb, {a_sk, get_var(pb, "zero_var")}, get_iv_add(pb), annotation_prefix)
```

Comment:
do we want to confirm the 0 value here ? (have two interface for get_var one having a 3rd argument being the value)